executable/microsoft_pe.ksy

```diff
@@ -16,6 +16,7 @@ meta:
   license: CC0-1.0
   ks-version: 0.7
```

Comment:
Also please change this to

```ksy
  ks-version: 0.9
```

... because little-endian bit-sized integers is a 0.9 feature. See https://doc.kaitai.io/user_guide.html#bit-ints-le:

> ### 4.14.2. Little-endian order
> 
> <table>
> <tr>
> <th>IMPORTANT</th>
> <td>Feature available since v0.9.</td>
> </tr>
> </table>

Reply:
Done